src/App.tsx

```diff
@@ -236,6 +322,54 @@ function App() {
             </InputRightElement>
           </InputGroup>
 
+          {/* TODO:
+          * * Test
+          * */}
+          <Heading mt={4} mb={1.5} size="sm">
+            Upload & Download 
+          </Heading>
+          <Text>
+            You can also upload with drag&drop and downlod with Ctrl + S
+          </Text>
+          <ButtonGroup 
+            size="sm"
+            display="flex"
+          >
+              <Button
+                size="sm"
+                colorScheme={darkMode ? "whiteAlpha" : "blackAlpha"}
+                borderColor={darkMode ? "purple.400" : "purple.600"}
+                color={darkMode ? "purple.400" : "purple.600"}
+                variant="outline"
+                leftIcon={<VscCloudUpload />}
+                mt={1}
+                flex="auto"
+                onClick={async () => {
+                  const file = await uploadFile();
+                  if (!file) return;
+                  handleUploadFile(file);
+                }}
+              >
+                Upload
+              </Button>
+              <Button
+                size="sm"
+                colorScheme={darkMode ? "whiteAlpha" : "blackAlpha"}
+                borderColor={darkMode ? "purple.400" : "purple.600"}
+                color={darkMode ? "purple.400" : "purple.600"}
+                variant="outline"
+                leftIcon={<VscCloudDownload />}
+                mt={1}
+                flex="auto"
+                onClick={(evt) => {
+                  evt.preventDefault();
+                  handleDownloadFile();
+                }}
+              >
+                Download
+              </Button>
```

Comment:
We should encapsulate this kind of button, which is the same as the "Read the Code" button, into a separate component.

Reply:
If we want to keep the styling between them the same.